src/test/scala/org/clulab/reach/TestModelEntities.scala

```diff
@@ -40,15 +39,13 @@ class TestModelEntities extends FlatSpec with Matchers {
     mentions.count(_ matches "Gene_or_gene_product") should be (4)
   }
 
-  // The following tests fail because of incorrect NER CRF identification as families
-  // See Processors issue #61
-  // "s2a entities" should "have GPP label" in {
-  //   val mentions = getBioMentions(s2a)
-  //   mentions.isEmpty should be (false)
-  //   printMentions(Try(mentions), true)      // DEBUGGING
-  //   mentions.size should be (4)
-  //   mentions.count(_ matches "Gene_or_gene_product") should be (4)
-  // }
+  "s2a entities" should "have GPP label" in {
+    val mentions = getBioMentions(s2a)
+    mentions.isEmpty should be (false)
```

Comment:
I think tests like this should be written as `mentions shouldBe empty` instead.
This way, if they fail, they can display an error message that is more useful for debugging.